templates/tonic-notifications-deployment.yaml

```diff
@@ -111,6 +111,12 @@ spec:
           value: "7000"
         - name: TONIC_NOTIFICATIONS_HEALTH_PORT_HTTPS
           value: "7001"
+        {{- if $server.envFrom }}
+        envFrom:
+        {{- with $server.envFrom }}
+            {{- toYaml . | nindent 8 }}
+        {{- end }}
+        {{- end }}
```

Comment:
Looks like the `$server` var was copied to all of the files

Reply:
yeah...  for some reason thought we had just set everything to server.  corrected